internal/cli/config/config_test.go

```diff
@@ -0,0 +1,122 @@
+package config
+
+import (
+	"os"
+	"sync"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNew_emptyEnv(t *testing.T) {
+	NewEnvTest(nil).Run(func() {
+		conf, err := New(nil)
+		require.EqualError(t, err, "invalid config settings: No token configured")
+		assert.Nil(t, conf)
+	})
+}
+
+func TestNew_minimalEnv(t *testing.T) {
+	NewEnvTest(map[string]string{
+		"UNMS_EXPORTER_TOKEN": "a=b",
+	}).Run(func() {
+		conf, err := New(nil)
+		require.NoError(t, err)
+		assert.EqualValues(t, &Config{
+			ServerAddr:   DefaultServerAddress,
+			LogLevel:     DefaultLogLevel,
+			TokenPerHost: tokenMap{"a": "b"},
+		}, conf)
+	})
+}
+
+func TestNew_multipleToken(t *testing.T) {
+	NewEnvTest(map[string]string{
+		"UNMS_EXPORTER_TOKEN": "a=b,c=d,e==f",
+	}).Run(func() {
+		conf, err := New(nil)
+		require.NoError(t, err)
+		assert.EqualValues(t, &Config{
+			ServerAddr: DefaultServerAddress,
+			LogLevel:   DefaultLogLevel,
+			TokenPerHost: tokenMap{
+				"a": "b",
+				"c": "d",
+				"e": "=f",
+			},
+		}, conf)
+	})
+}
+
+func TestNew_flagsTakePriorityOverEnv(t *testing.T) {
+	NewEnvTest(map[string]string{
+		"UNMS_EXPORTER_TOKEN":  "a=b",
+		"UNMS_EXPORTER_LISTEN": "::1",
```

Comment:
although this is a valid string for the config, it would not succeed when starting the service, since ipv6 addresses need to be enclosed in brackets and the listen address needs to include a port.
consider using `[::1]:0` as an example instead.

this also applies to other tests and the testdata

Reply:
Done.